tee-worker/omni-executor/rpc-server/src/methods/pumpx/add_wallet.rs

```diff
@@ -0,0 +1,57 @@
+use crate::{
+	error_code::*, hex_encode, oneshot, server::RpcContext, verify_auth::verify_auth, Deserialize,
+	ErrorCode,
+};
+use executor_core::native_task::*;
+use executor_primitives::OmniAuth;
+use heima_primitives::{Identity, Web2IdentityType};
+use jsonrpsee::{types::ErrorObject, RpcModule};
+
+#[derive(Debug, Deserialize)]
+pub struct AddWalletParams {
+	pub user_email: String,
+	pub auth_token: String,
+}
+
+impl From<AddWalletParams> for NativeTaskWrapper<NativeTask> {
+	fn from(p: AddWalletParams) -> Self {
+		Self {
+			task: NativeTask::PumpxAddWallet(Identity::from_web2_account(
```

Comment:
How do you ensure both `user_email` and `auth_token` relates to same `Identity` ?

If you have a JWT token can I impersonate you by specifying your email here and using my JWT token ?